propono.gemspec

```diff
@@ -20,6 +20,7 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency "aws-sdk-sns"
   spec.add_dependency "aws-sdk-sqs"
+  spec.add_dependency "nokogiri"
```

Comment:
Why do we need this?

Reply:
Very annoying I agree. For Ruby 3, I got error messages from the AWS libraries indicating that an XML parser was required. 

```
/Users/adavis/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/aws-sdk-core-3.125.4/lib/aws-sdk-core/xml/parser.rb:74:in `set_default_engine': Unable to find a compatible xml library. Ensure that you have installed or added to your Gemfile one of ox, oga, libxml, nokogiri or rexml (RuntimeError)
```

Reply:
This is actually kind of neat on the part of AWS. Their library needs an XML parser, but your project probably already has one of the common ones, so support all of the common ones.

There is no nice way to downstream that for `propono` users though, without a lot of explanation.

Reply:
The error message was ace. I dream of writing messages that useful to my users

Reply:
I have updated the gemspec to make nokogiri a development dependency